From 7c6adc03fa1c1ef6cf6f71451480110957a5713f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=B2=88=E5=AD=90=E4=BF=8A?= <shenzijun@kylinos.cn>
Date: Mon, 27 Dec 2021 19:41:33 +0800
Subject: [PATCH] Add the configuration for accelerated of SM4
MIME-Version: 1.0
Patch-mainline: NO, OTHERS
References: OLK-5.10

Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

kylin inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4A842?from=project-issue
CVE: NA

-------------------------------------------------------------------------

Add the configuration in arch/x86/configs/openeuler_defconfig

Signed-off-by: 沈子俊 <shenzijun@kylinos.cn>
Reviewed-by: weiyang wang <wangweiyang2@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 7b608301823c..e0dc1686984d 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -8001,6 +8001,8 @@ CONFIG_CRYPTO_SERPENT_SSE2_X86_64=m
 CONFIG_CRYPTO_SERPENT_AVX_X86_64=m
 CONFIG_CRYPTO_SERPENT_AVX2_X86_64=m
 CONFIG_CRYPTO_SM4=m
+CONFIG_CRYPTO_SM4_AESNI_AVX_X86_64=m
+CONFIG_CRYPTO_SM4_AESNI_AVX2_X86_64=m
 CONFIG_CRYPTO_TEA=m
 CONFIG_CRYPTO_TWOFISH=m
 CONFIG_CRYPTO_TWOFISH_COMMON=m
-- 
2.26.2

